tests/integration-tests/tests/storage/snapshots_factory.py

```diff
@@ -0,0 +1,286 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import logging
+import threading
+import time
+from collections import namedtuple
+
+import boto3
+from retrying import retry
+
+from fabric import Connection
+
+SnapshotConfig = namedtuple("ClusterConfig", "ssh_key key_name vpc_id master_subnet_id")
```

Comment:
minor: you could use the new version of namedtuple as done in the other factories.